### DISCUSSION TITLE
Creating a PDF based on the submission

### DISCUSSION THREAD

**Discussion by luismgarcia** [tobimori/kirby-dreamform#20] [Feature Requests]
Creating a PDF based on the submission

**Comment by tobimori** [comment]
We've implemented this solution for a client together with the help of another Kirby Partner. It will not make it into DreamForm because it requires a Headless Browser to render the document and this is not available on most hosting solutions, however if this is a strict requirement for your project, it's definitely possible and we can talk about developing a custom solution.
